functional_tests.go

```diff
@@ -2623,6 +2623,22 @@ func testCopyObject() {
 		return
 	}
 
+	// Source object
+	r, err = c.GetObject(bucketName, objectName, minio.GetObjectOptions{})
+	if err != nil {
+		logError(testName, function, args, startTime, "", "GetObject failed", err)
+		return
+	}
+
+	// Get correct object info
+	objInfo, err = r.Stat()
+	if err != nil {
+		logError(testName, function, args, startTime, "", "Stat failed", err)
+		return
+	}
+
+	r.Close()
+
 	stOpts := minio.StatObjectOptions{}
 	stOpts.SetMatchETag(objInfo.ETag)
```

Comment:
If this is required why is it working fine with server mode?

Reply:
Looks like [CopyObject to update metadata](https://github.com/minio/minio-go/blob/master/functional_tests.go#L2620) updates the Etag for Azure , while for Minio Etag doesn't change

Reply:
I see